.gitignore

```diff
@@ -123,4 +123,39 @@ env/
 .env
 
 ### Logs ###
-geckodriver.log
\ No newline at end of file
+geckodriver.log
+
+# Local .terraform directories
+**/.terraform/*
```

Comment:
You may be conscious already as you've used the right syntax here that works for both, but [worth noting that there are some subtle differences between `.gitignore` and `.dockerignore`](https://zzz.buzz/2018/05/23/differences-of-rules-between-gitignore-and-dockerignore/) that means duplicating arguments between them can lead to missing some things